### ISSUE TITLE
Problem with the example

### ISSUE THREAD

**Issue by prewk** [shrhdk/text-to-svg#2]
Hi, I tried running the example:

````js
var TextToSVG = require('text-to-svg').TextToSVG;
var textToSVG = new TextToSVG();

var attributes = {fill: 'red', stroke: 'black'};
var options = {x: 0, y: 0, fontSize: 72, attributes: attributes};

var svg = textToSVG.getSVG('hello', options);

console.log(svg);
````

I saved the output to a file:

````svg
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path fill="red" stroke="black" d="M 5.2734375 -54.0703125 L 10.6171875 -54.0703125 L 10.6171875 -33.99609375 Q 15.85546875 -39.234375 21.0234375 -39.234375 Q 26.89453125 -39.234375 29.6015625 -34.06640625 Q 31.11328125 -31.1484375 31.11328125 -27 L 31.11328125 -3.65625 L 25.76953125 -3.65625 L 25.76953125 -25.41796875 Q 25.76953125 -34.13671875 20.1796875 -34.13671875 Q 16.41796875 -34.13671875 13.5703125 -31.39453125 Q 10.6171875 -28.44140625 10.6171875 -24.64453125 L 10.6171875 -3.65625 L 5.2734375 -3.65625 L 5.2734375 -54.0703125 Z M 67.67578125 -14.2734375 Q 64.546875 -2.25 54.0703125 -2.25 Q 47.56640625 -2.25 43.76953125 -7.6640625 Q 40.32421875 -12.62109375 40.32421875 -20.7421875 Q 40.32421875 -28.51171875 43.55859375 -33.46875 Q 47.35546875 -39.234375 54 -39.234375 Q 66.97265625 -39.234375 67.81640625 -19.65234375 L 45.73828125 -19.65234375 Q 46.16015625 -7.06640625 54.140625 -7.06640625 Q 60.46875 -7.06640625 62.05078125 -14.2734375 L 67.67578125 -14.2734375 M 62.05078125 -24.2578125 Q 60.890625 -34.41796875 54 -34.41796875 Q 47.35546875 -34.41796875 45.94921875 -24.2578125 L 62.05078125 -24.2578125 Z M 92.8125 -11.53125 Q 92.8125 -8.4375 95.765625 -8.4375 Q 98.19140625 -8.4375 101.07421875 -9 L 101.07421875 -3.62109375 Q 96.8203125 -3.0234375 94.81640625 -3.0234375 Q 87.1875 -3.0234375 87.1875 -10.51171875 L 87.1875 -54.0703125 L 92.8125 -54.0703125 L 92.8125 -11.53125 Z M 128.8125 -11.53125 Q 128.8125 -8.4375 131.765625 -8.4375 Q 134.19140625 -8.4375 137.07421875 -9 L 137.07421875 -3.62109375 Q 132.8203125 -3.0234375 130.81640625 -3.0234375 Q 123.1875 -3.0234375 123.1875 -10.51171875 L 123.1875 -54.0703125 L 128.8125 -54.0703125 L 128.8125 -11.53125 Z M 162.0703125 -39.234375 Q 168.6796875 -39.234375 172.44140625 -33.3984375 Q 175.67578125 -28.546875 175.67578125 -20.7421875 Q 175.67578125 -14.87109375 173.7421875 -10.44140625 Q 170.15625 -2.21484375 161.9296875 -2.21484375 Q 155.56640625 -2.21484375 151.76953125 -7.62890625 Q 148.32421875 -12.5859375 148.32421875 -20.7421875 Q 148.32421875 -29.53125 152.296875 -34.55859375 Q 156.09375 -39.234375 162.0703125 -39.234375 M 161.9296875 -34.20703125 Q 158.0625 -34.20703125 155.8828125 -30.1640625 Q 153.94921875 -26.61328125 153.94921875 -20.7421875 Q 153.94921875 -15.328125 155.53125 -11.91796875 Q 157.7109375 -7.2421875 162 -7.2421875 Q 165.9375 -7.2421875 168.1171875 -11.28515625 Q 170.05078125 -14.8359375 170.05078125 -20.671875 Q 170.05078125 -26.75390625 168.046875 -30.234375 Q 165.90234375 -34.20703125 161.9296875 -34.20703125 Z"/></svg>
````

But it doesn't seem to work, it's just white when I open it in Chrome/Safari or trying to preview it in Finder. Am I misunderstanding how it should work?

**Comment by prewk** [comment]
Hm, it seems it's only offset, increasing `y` made my text show up. Is this intended? It would be great if 0, 0 would render from upper left to lower right!

**Comment by shrhdk** [comment]
I know your thought well :+1: 

I think that a solution is anchor point option.
The current anchor point is `left-bottom`, but you expected `left-top`.

I will consider to add anchor point option.

**Comment by prewk** [comment]
Yeah, it would be great if you could render the text and always be sure that everything rendered is visible. I'm experimenting with some kind of "subtract lowest X and Y on the coords" algorithm without knowing anything about SVG, really.. That might be a workaround as well, to trim the whitespace.

Thanks for the lib btw, only one I've found which doesn't require `node-canvas`, which in turn requires pre-installed C libs, which isn't very convenient when trying to render stuff in a CI process or on many dev's computers!

**Comment by shrhdk** [comment]
I developed this lib for the same reason as you. I am happy to help you.

I released version `2.1.0` which supports `anchor` option.

## Sample Code

```js
var TextToSVG = require('text-to-svg').TextToSVG;
var textToSVG = new TextToSVG();

var svg = textToSVG.getSVG('hello', {anchor: "top"});

console.log(svg);
```

## Result

```svg
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path d="M 5.2734375 9.28125 L 10.6171875 9.28125 L 10.6171875 29.35546875 Q 15.85546875 24.1171875 21.0234375 24.1171875 Q 26.89453125 24.1171875 29.6015625 29.28515625 Q 31.11328125 32.203125 31.11328125 36.3515625 L 31.11328125 59.6953125 L 25.76953125 59.6953125 L 25.76953125 37.93359375 Q 25.76953125 29.21484375 20.1796875 29.21484375 Q 16.41796875 29.21484375 13.5703125 31.95703125 Q 10.6171875 34.91015625 10.6171875 38.70703125 L 10.6171875 59.6953125 L 5.2734375 59.6953125 L 5.2734375 9.28125 Z M 67.67578125 49.078125 Q 64.546875 61.1015625 54.0703125 61.1015625 Q 47.56640625 61.1015625 43.76953125 55.6875 Q 40.32421875 50.73046875 40.32421875 42.609375 Q 40.32421875 34.83984375 43.55859375 29.8828125 Q 47.35546875 24.1171875 54 24.1171875 Q 66.97265625 24.1171875 67.81640625 43.69921875 L 45.73828125 43.69921875 Q 46.16015625 56.28515625 54.140625 56.28515625 Q 60.46875 56.28515625 62.05078125 49.078125 L 67.67578125 49.078125 M 62.05078125 39.09375 Q 60.890625 28.93359375 54 28.93359375 Q 47.35546875 28.93359375 45.94921875 39.09375 L 62.05078125 39.09375 Z M 92.8125 51.8203125 Q 92.8125 54.9140625 95.765625 54.9140625 Q 98.19140625 54.9140625 101.07421875 54.3515625 L 101.07421875 59.73046875 Q 96.8203125 60.328125 94.81640625 60.328125 Q 87.1875 60.328125 87.1875 52.83984375 L 87.1875 9.28125 L 92.8125 9.28125 L 92.8125 51.8203125 Z M 128.8125 51.8203125 Q 128.8125 54.9140625 131.765625 54.9140625 Q 134.19140625 54.9140625 137.07421875 54.3515625 L 137.07421875 59.73046875 Q 132.8203125 60.328125 130.81640625 60.328125 Q 123.1875 60.328125 123.1875 52.83984375 L 123.1875 9.28125 L 128.8125 9.28125 L 128.8125 51.8203125 Z M 162.0703125 24.1171875 Q 168.6796875 24.1171875 172.44140625 29.953125 Q 175.67578125 34.8046875 175.67578125 42.609375 Q 175.67578125 48.48046875 173.7421875 52.91015625 Q 170.15625 61.13671875 161.9296875 61.13671875 Q 155.56640625 61.13671875 151.76953125 55.72265625 Q 148.32421875 50.765625 148.32421875 42.609375 Q 148.32421875 33.8203125 152.296875 28.79296875 Q 156.09375 24.1171875 162.0703125 24.1171875 M 161.9296875 29.14453125 Q 158.0625 29.14453125 155.8828125 33.1875 Q 153.94921875 36.73828125 153.94921875 42.609375 Q 153.94921875 48.0234375 155.53125 51.43359375 Q 157.7109375 56.109375 162 56.109375 Q 165.9375 56.109375 168.1171875 52.06640625 Q 170.05078125 48.515625 170.05078125 42.6796875 Q 170.05078125 36.59765625 168.046875 33.1171875 Q 165.90234375 29.14453125 161.9296875 29.14453125 Z"/></svg>
```

**Comment by prewk** [comment]
Awesome, I'll try it out! Thanks!
